aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsFluentClientDecorator.kt

```diff
@@ -82,9 +82,10 @@ class AwsFluentClientDecorator : RustCodegenDecorator {
             AwsFluentClientExtensions(types).render(writer)
         }
         val awsHyper = "aws-hyper"
-        rustCrate.mergeFeature(Feature("client", default = true, listOf(awsHyper, "aws-smithy-client")))
-        rustCrate.mergeFeature(Feature("rustls", default = true, listOf("$awsHyper/rustls")))
-        rustCrate.mergeFeature(Feature("native-tls", default = false, listOf("$awsHyper/native-tls")))
+        val awsSmithyClient = "aws-smithy-client"
+        rustCrate.mergeFeature(Feature("client", default = true, listOf(awsSmithyClient, awsHyper)))
```

Comment:
Was the use of crate names instead of feature names intentional in the feature list for `client`?

@rcoh - What is the use-case for the `client` feature? Is this something we could potentially remove?

Reply:
I figured we could remove it but I wanted to ask first since I do think it could affect the Server folks

Reply:
the use case is that you could get a SDK that didn't actually depend on anything (just included the operations)